tests/integration/run-locally.sh

```diff
@@ -3,19 +3,22 @@
 set -e
 
 expandpath() {
-  CDPATH= cd -- "$1" && pwd -P
+  CDPATH=$(cd -- "$1" && pwd -P)
+  echo "$CDPATH"
 }
```

Comment:
Yep, I fix a warning by making the parenthesis also include the second clause, which was wrong. 
If I keep:
```CDPATH=cd -- "$1" && pwd -P```  
I get the warning from shellcheck:
 `To assign the output of a command, use var=$(cmd) . [SC2037]`
but I follow the pattern:
`CDPATH=$(cd -- "$1") && pwd -P`
I don't get cd to the script's directory (probably because it runs in a subshell).

Edge case that can't cover shellcheck?